content/views.py

```diff
@@ -117,3 +117,103 @@ def get(self, request, content_id):
             return JsonResponse({'MESSAGE' : 'SUCCESS', 'RESULT' : results}, status=200)
         except Content.DoesNotExist:
             return JsonResponse({'MESSAGE' : 'INVALID_CONTENT_ID'}, status=400)
+
+
+class ContentOverview(View):
+    def get(self, request, content_id):
+        try:
+            content          = Content.objects.get(id=content_id)
+
+            result = {
+                'content_id'     : content.id,
+                'title_original' : content.movieoverview_set.get().title_original,
+                'release_year'   : content.release_year,
+                'genre'          : [genre.genre.name for genre in content.contentgenre_set.all()],
+                'runtime'        : content.movieoverview_set.get().runtime,
+                'description'    : content.movieoverview_set.get().description,
+            }
+            return JsonResponse({'MESSAGE' : 'SUCCESS', 'RESULT' : result}, status=200)
+        except Content.DoesNotExist:
+            return JsonResponse({'MESSAGE' : 'INVALID_CONTENTS_ID'}, status=400)
+
+
+class ContentCast(View):
+    def get(self, request, content_id):
+        try:
+            content = Content.objects.get(id=content_id)
+
+            cast_list = [
+                {
+                    'people_id'        : people.people.id,
+                    'people_name'      : people.people.name,
+                    'role'             : people.role_name,
+                    'people_image_url' : people.people.profile_image_url,
+                } for people in content.contentpeople_set.all()
+            ]
+
+            results = {
+                'cast' : cast_list,
+            }
```

Comment:
이 부분은 전달이 잘 못 된 것 같아요!
```suggestion
            results = [
                {
                    'people_id'        : people.people.id,
                    'people_name'      : people.people.name,
                    'role'             : people.role_name,
                    'people_image_url' : people.people.profile_image_url,
                } for people in content.contentpeople_set.all()
            ]
```